spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
What are your thoughts - which is better and why? 
Should we not be returning an array of partners because we are stubbing out the paginate method. Calling paginate on a return value of paginate, to me, seems a little bit misleading.

```
specify 'pagination' do
  partner = build_stubbed :partner
  expect(Partner).to receive_message_chain(:all, :paginate).with(page:
  "2").and_return([partner])
  get :index, page: "2"
end
```